.github/workflows/ci.yml

```diff
@@ -438,12 +438,7 @@ jobs:
     name: "Wait for CI images"
     runs-on: ["ubuntu-22.04"]
     needs: [build-info, build-ci-images]
-    # This strange condition below is equivalent to: "all success or skipped"
```

Comment:
Unfortunately - this does not work as I thought. The "skip" status propagates to downstream jobs and if we skip "build-ci-images" even if "wait-for-ci-images" successfully runs with this condition, downstream jobs are still skipped :(